pyproject.toml

```
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "brush"
+authors = [{ name = "Xiaokang2022" }, { email = "2951256653@qq.com" }]
+maintainers = [{ name = "Xiaokang2022" }, { email = "2951256653@qq.com" }]
+description = "Official designer software of `maliang` for visualize development."
```

```suggestion
description = "Official designer software of maliang for visualize development."
```
这个反引号就不需要了，不是在 README 和 docstring 里面，PyPI 识别不了

---

长见识了，我没发过PyPI